sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -205,7 +213,7 @@ class CodeGenContext {
 
 
 abstract class GeneratedClass {
```

Comment:
while you are at this, can you add scaladoc for GeneratedClass? It is not obvious what it does just by looking at it.